clinica/pydra/statistics_volume/template_model_contrast.m

```diff
@@ -0,0 +1,21 @@
+% This file has been obtained with SPM
+% Launch SPM menu > Stats > Contrast Manager
+% Then View > Show m-code
+
+spm('defaults','pet');
```

Comment:
If these matlab files are exactly the same as for nipype, then we shouldn't copy them. We should just modify the links to point to the Nipype folder instead.